src/targets/commitOnGitRepository.ts

```diff
@@ -153,7 +150,8 @@ export async function pushArchiveToGitRepository({
 
       const authenticatedUrl = parsedUrl.toString();
 
-      await git.clone(authenticatedUrl, directory);
+      await createGitClient('.').clone(authenticatedUrl, directory);
```

Comment:
Same as earlier: shouldn't we use `createGitClient(directory)` instead of `createGitClient('.')`